aesara/tensor/random/basic.py

```diff
@@ -1244,6 +1244,55 @@ def rng_fn_scipy(cls, rng, b, loc, scale, size):
 truncexpon = TruncExponentialRV()
 
 
+class StudentTRV(ScipyRandomVariable):
+    r"""A Student's t continuous random variable.
+
+    The probability density function for `t` in terms of its degrees of freedom
+    parameter :math:`\nu`, location parameter :math:`\mu` and scale
+    parameter :math:`\sigma` is:
+
+    .. math::
+
+        f(x; \nu, \alpha, \beta) = \frac{\Gamma(\frac{\nu + 1}{2})}{\Gamma(\frac{\nu}{2})} \left(\frac{1}{\pi\nu\sigma}\right)^{\frac{1}{2}} \left[1+\frac{(x-\mu)^2}{\nu\sigma}\right]^{-\frac{\nu+1}{2}}
+
+    for :math:`\nu > 0`, :math:`\sigma > 0`.
+
+    """
+    name = "t"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("StudentT", "\\operatorname{StudentT}")
+
+    def __call__(self, df, loc=0.0, scale=1.0, size=None, **kwargs):
+        r"""Draw samples from a Student's t distribution.
+
+        Parameters
+        ----------
+        df
+            Degrees of freedom parameter :math:`\nu` of the distribution. Must be
+            positive.
+        loc
+            Location parameter :math:`\mu` of the distribution.
+        scale
+            Scale parameter :math:`\sigma` of the distribution. Must be
+            positive.
+        size
+           Sample shape. If the given size is `(m, n, k)`, then `m * n * k`
+           independent, identically distributed samples are returned. Default is
+           `None` in which case a single sample is returned.
+
+        """
+        return super().__call__(df, loc, scale, size=size, **kwargs)
+
+    @classmethod
+    def rng_fn_scipy(cls, rng, df, loc, scale, size):
+        return stats.t.rvs(df, loc=loc, scale=scale, size=size, random_state=rng)
+
+
+t = StudentTRV()
```

Comment:
We should at least expose this via our NumPy-like API (e.g. create a `standard_t`).

Reply:
I did this to be consistent with  `Cauchy` and `Gamma` that are implemented as `ScipyRandomVariable`s even though there is a `standard_cauchy` and `standard_gamma` in NumPy. I don't mind either way.

Reply:
> Oh did you mean adding `standard_t`? In this case I can add `standard_cauchy`, `standard_exponential` and `standard_gamma` for NumPy compatibility as well while I'm at it.

Yeah, that would be good.